Ecommerce/app/models/category.rb

```diff
@@ -1,10 +1,6 @@
 class Category < ApplicationRecord
-    has_many :items
-
-    def self.ransackable_attributes(auth_object = nil)
-      ["created_at", "id", "name", "updated_at"]
-    end
-
-    enum name: { Male: 0, Female: 1}
-  end
+  include Ransackable
+  has_many :items    
```

Comment:
We leave an extra line to differentiate where associations are ending, where delegates are starting where enum will start.
So don't give extra line while adding same kind of things but leave an extra line when different section is starting